draft-murillo-whep.md

```diff
@@ -72,37 +72,48 @@ Once the ICE/DTLS session is set up, the media will flow unidirectionally from M
 ~~~~~
 {: title="WHEP session setup and teardown"}
 
+
+Alternatively, there are cases in which the WHEP player may wish the service to provide the SDP offer (for example to avoid seting up an audio and video session when only audio is supported), so in this case the initial HTTP POST request will not contain a body and the response will contain the SDP offer from the service instead. The WHEP player will have to provide the SDP answer in a subsequent HTTP PATCH request to the WHEP resource.
+~~~~~
+                                                                               
+ +-------------+    +---------------+ +--------------+ +---------------+
+ | WHEP player |    | WHEP endpoint | | Media Server | | WHEP Resource |
+ +--+----------+    +---------+-----+ +------+-------+ +--------|------+
+    |                         |              |                  |       
+    |                         |              |                  |       
+    |HTTP POST (empty)        |              |                  |       
+    +------------------------>+              |                  |       
+    |201 Created (SDP offer)  |              |                  |       
+    +<------------------------+              |                  | 
```

Comment:
The 201 could return also the list of ice servers instead of requiring an additional HTTP request to retrieve them.

It is less elegant than the current approach but also faster, so maybe it is worth considering it.

From an implementer point of view is very simple to do this:
```
const { iceServers, offer } = await fetch(resourceUrl)
const pc = new RTCPeerConnection({ iceServers }}
await pc.setRemoteDescription({ type: offer, sdp: offer })
```

Reply:
yes, the 201 will return the ice servers regardless if the post has an sdp offer or it is empty